CoreLibs/nbproject/project.xml

```diff
@@ -970,6 +971,10 @@
                 <runtime-relative-path>ext/gstreamer-java-1.5.jar</runtime-relative-path>
                 <binary-origin>release/modules/ext/gstreamer-java-1.5.jar</binary-origin>
             </class-path-extension>
+            <class-path-extension>
+                <runtime-relative-path>ext/dd-plist-1.20.jar</runtime-relative-path>
+                <binary-origin>C:\Users\kelly\Workspace\autopsy\CoreLibs\release\modules\ext\dd-plist-1.20.jar</binary-origin>
```

Comment:
Here is a hard-coded path specific to your machine.

Reply:
fixed